src/artemis/main.py

```diff
@@ -17,88 +25,131 @@
 class Actions(Enum):
     START = "start"
     STOP = "stop"
+    SHUTDOWN = "shutdown"
 
 
 class Status(Enum):
     FAILED = "Failed"
     SUCCESS = "Success"
     BUSY = "Busy"
+    ABORTING = "Aborting"
     IDLE = "Idle"
 
 
+@dataclass
+class Command:
+    action: Actions
+    parameters: Optional[FullParameters] = None
+
+
+@dataclass_json
 @dataclass
 class StatusAndError:
```

Comment:
Maybe rename this class to StatusAndMessage or something similar? Putting Error in the class name is a little misleading at a glance, because if I see an object being created with "Error" in it I assume an error/exception is being raised rather than a status being reported.